src/stories/Values.stories.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { Values } from 'blocks'
+import aboutUsData from '../../data/about-us'
+
+export default {
+  title: 'Blocks/Values',
+}
+
+export const Default = () => {
```

Comment:
Out of curiosity, why `export const Default` and not `export default` ?

Reply:
Because we already have the following above 😉 : 
```js
export default {
  title: 'Blocks/Values',
}
```

Plus, this way it appears like this on the Storybook front-end: 
<img width="216" alt="Capture d’écran 2022-11-14 à 09 07 19" src="https://user-images.githubusercontent.com/30866152/201607654-2b02dddd-c78d-49a2-8a9c-236cf81b6829.png">

And that's what we want! :D